src/Catalog/Schema.cs

```diff
@@ -153,6 +153,13 @@ public static class Predicates
             public static readonly Uri LicenseFile = new Uri(Prefixes.NuGet + "licenseFile");
 
             public static readonly Uri IconFile = new Uri(Prefixes.NuGet + "iconFile");
+
+            public static readonly Uri Deprecation = new Uri(Prefixes.NuGet + "deprecation");
+            public static readonly Uri DeprecationReasons = new Uri(Prefixes.NuGet + "reasons");
+            public static readonly Uri DeprecationMessage = new Uri(Prefixes.NuGet + "message");
+            public static readonly Uri DeprecationAlternatePackage = new Uri(Prefixes.NuGet + "alternatePackage");
+            public static readonly Uri DeprecationAlternatePackageId = new Uri(Prefixes.NuGet + "id");
+            public static readonly Uri DeprecationAlternatePackageVersion = new Uri(Prefixes.NuGet + "version");
```

Comment:
This is a version range not version, right?

Reply:
Yes, but the field is titled "version", which is why I chose this instead of "VersionRange" or similar.

Reply:
Hm... okay. Sounds good.

Reply:
Note: I looked at existing catalog pages and saw that dependency version ranges have `range` as the property name instead of `version`. Do you think I should switch to that syntax? I think it would improve the readability of the code.

Reply:
Well, there is the RDF answer and the more down to earth answer for what we want our JSON to look like. You could make arguments either way. I personally think it's wrong to come up with new names for the same thing, but you could argue this is a different thing since package dependencies should not allow `*` but deprecation does.

In short, I think `range` is a more consistent name.

Reply:
switching to `range`